contracts/hardhat.config.ts

```diff
@@ -19,7 +19,7 @@ const config: HardhatUserConfig = {
   },
   networks: {
     devnet: {
-      url: "http://127.0.0.1:5050/",
+      url: 'http://127.0.0.1:5050/',
```

Comment:
All other places use double quotes, so we should stick to that. I added an issue to unify styling: https://github.com/eqlabs/starknet-multisig/issues/23